ui-v2/app/components/healthchecked-resource.js

```diff
@@ -1,15 +1,16 @@
+import { filter } from '@ember/object/computed';
```

Comment:
Hmm, not massively sure I wanted it to do this 🤔 . I suppose its looking forwards to tree shaking, but I'm already importing computed for another thing further down, so I kinda like using the namespaced `computed.filter` 🤷‍♂ 

Not gonna change anything here, but was interesting nonetheless